examples/advertiser.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	"time"
+
+	"github.com/RadiusNetworks/go-beacon"
+	"github.com/RadiusNetworks/go-beacon/advertiser"
+)
+
+func main() {
+	urlBeacon, _ := beacon.NewEddystoneURLBeacon("https://radiusnetworks.com", -42)
+	eddystoneURLParser := beacon.NewParser("eddystone_url", beacon.DefaultLayouts["eddystone_url"])
+	advert := eddystoneURLParser.GenerateAd(urlBeacon)
+	adv, _ := advertiser.New()
+	for {
+		adv.AdvertiseMfgData(0xfeaa, advert)
+		time.Sleep(5 * time.Second)
+	}
```

Comment:
I don't think you need to call `AdvertiseMfgData` in a loop.  I think once you set it, it will advertise on whatever interval the hardware is set for.

Probably what you need to do after calling `AdvertiseMfgData` is wait on an os signal (sigterm or sigint) and then when received call `StopAdvertising` and then exit.  I think this is how you'd wait on sigterm or sigint:
```go
sigChan := make(chan os.Signal, 1)
signal.Notify(sigChan, syscall.SIGINT)
signal.Notify(sigChan, syscall.SIGTERM)
<-sigChan // wait for signal
```